03/app.js

```diff
@@ -1,18 +1,60 @@
-const min = 1;
-const max = 100;
-
-const a = randomNumber(min, max);
-const b = randomNumber(min, max);
-const c = randomNumber(min, max);
-
-console.log(a, b, c);
-
-
-
-
+const min = 1
+const max = 100
 
+const a = randomNumber(min, max)
+const b = randomNumber(min, max)
+const c = randomNumber(min, max)
 
+console.log(a, b, c)
 
 function randomNumber(min, max) {
-    return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+	return Math.round(Math.random() * (max - min) + min)
+}
+
+function getSum(a, b, c) {
+	if (a > b && c > b) {
+		return a + c
+	}
+	if (b > a && c > a) {
+		return b + c
+	}
+	if (a > c && b > c) {
+		return a + b
+	}
+	if (a === b && a === c && b === c) {
+		console.log(`wygenerowane liczby są takie same pierwsza=${a},druga=${b},trzecia=${c}`)
+	}
```

Comment:
W zadaniu było zwrócić 2 największe - jeśli są równe to po prostu zwracamy obojętnie, które, ale nadal zachowujemy sposób działania jaki był określony :) Inaczej inny programista może nie wiedzieć o co chodzi ;)